src/MaIN.Domain/Entities/Agents/Knowledge/KnowledgeIndexItemSlim.cs

```diff
@@ -0,0 +1,8 @@
+namespace MaIN.Domain.Entities.Agents.Knowledge;
+
+public class KnowledgeIndexItemSlim
```

Comment:
Is this class used anywhere? Only usage is in KnowledgeIndexCheckResult which is not used anywhere